common-controller/internal/operator/controller/ratelimitpolicy_controller.go

```diff
@@ -136,14 +136,17 @@ func (ratelimitReconsiler *RateLimitPolicyReconciler) Reconcile(ctx context.Cont
 		return ctrl.Result{}, nil
 	}
 	logger.Info("add api ratelimit")
-	var vhost, resolveRatelimit = ratelimitReconsiler.marshelRateLimit(ctx, ratelimitKey, ratelimitPolicy)
+	var vhost, resolveRatelimit, customRateLimitPolicy = ratelimitReconsiler.marshelRateLimit(ctx, ratelimitKey, ratelimitPolicy)
```

Comment:
shall we move this to new method for keep the code clean

Reply:
ack

Reply:
fixed